crates/pixi_command_dispatcher/src/command_dispatcher/mod.rs

```diff
@@ -653,7 +657,11 @@ impl CommandDispatcherData {
     ///
     /// This function does not check if the path exists and also does not follow
     /// symlinks.
-    fn resolve_typed_path(&self, path_spec: Utf8TypedPath) -> Result<PathBuf, InvalidPathError> {
+    fn resolve_typed_path(
+        &self,
+        path_spec: Utf8TypedPath,
+        alternative_root_path: Option<&Path>,
```

Comment:
Lets call this `alternative_root` to stay in the same language as the `root_dir`.